tox.ini

```diff
@@ -42,6 +42,7 @@ deps =
     datadog==0.22.0
     prometheus-client==0.5.0
     dropbox: dropbox>=9.3.0<10.0
+    tenacity>=6.3.0<7    
```

Comment:
```suggestion
    tenacity>=6.3.0,<7
```

Reply:
Is this change needed? Shouldn't be this pulled in by `install_requires` in `setup.py`?

Reply:
This is unnecessary. Fixed!